lib/xcpretty/formatters/formatter.rb

```diff
@@ -173,10 +174,13 @@ def format_will_not_be_code_signed(message)
     end
 
     def format_other(text)
-      ""
+      if @unmatched and text and text.empty? == false and text.gsub(/\s+/, '').lstrip.empty? == false #and text != "\n" 
+        "unmatched: " + text.rstrip
```

Comment:
Why only `.rstrip`? Do you intentionally leave leading whitespace? If so, why? If not, what am I missing?

Reply:
if multiple "non matching" lines are printed and indented, I prefer to keep their indentation for readability